README.md

```diff
@@ -450,6 +454,65 @@ stdio to an in-memory [virtual terminal](https://github.com/hinshun/vt10x).
 
 For some examples, you can see any of the tests in this repo.
 
+### Using a mock
+
+Instead of calling the survey functions directly, you can create a survey struct and call the functions from there.
+
+```golang
+
+survey := survey.Survey{}
```

Comment:
This shadows the `survey` package with a `survey` variable. It's perfectly valid Go, but it might confuse the reader, since past this point `survey` isn't what it was before. For clarity, could we call the struct something different? I'm also thinking whether there would be a better name for the Survey struct (for the same reason)

Reply:
I noticed this while writing, but i couldn't come up with a better name :(

Any suggestions?

Reply:
How about "Surveyer"? Keeps the name of the library but is still not exactly the same.

Reply:
That sounds great

Reply:
I implemented it, but changed to "Surveyor" (does not look like a spelling mistake that way 😄)